src/components/grid_popover/grid_popover.ts

```diff
@@ -9,13 +10,32 @@ interface Props {
   onClosePopover: () => void;
   onMouseWheel: (ev: WheelEvent) => void;
 }
+
 export class GridPopover extends Component<Props, SpreadsheetChildEnv> {
   static template = "o-spreadsheet-GridPopover";
   static components = { Popover };
 
+  menuService = useState<MenuService>(this.env.menuService);
+  private wasMenuOpen = this.menuService.hasOpenMenu();
+
+  setup() {
+    onPatched(() => {
```

Comment:
`onWillUpdateProps` to avoid a useless rendering ? 
`onWillUpdateProps` is called *before* while `onPatched` is called *after*

Reply:
Dispatching a command in `onWillUpdateProps` makes everything crash :x